web/src/params/coaching_session.rs

```diff
@@ -1,15 +1,64 @@
 use chrono::{NaiveDate, NaiveDateTime};
+use domain::coaching_session::CoachingSessionSortParams;
+use domain::coaching_sessions;
 use domain::Id;
 use domain::{IntoQueryFilterMap, IntoUpdateMap, QueryFilterMap, UpdateMap};
-use sea_orm::Value;
+use sea_orm::{Order, Value};
 use serde::Deserialize;
 use utoipa::{IntoParams, ToSchema};
 
+#[derive(Debug, Deserialize, utoipa::ToSchema)]
+#[schema(example = "date")]
+pub(crate) enum CoachingSessionSortField {
```

Comment:
Can we chance this to just `SortField` since the `CoachingSession` part will be encoded in the module name `..params::coaching_session::SortField`

Reply:
Done.